src/http/client.cr

```diff
@@ -13,6 +13,20 @@
 # response.body.lines.first # => "<!doctype html>"
 # ```
 #
+# ### Parameters
+#
+# Parameters can be added to any request with the `HTTP::Params#encode` method, which
+# converts a `Hash` or `NamedTuple` to a URL encoded HTTP query.
+#
+# ```
+# require "http/client"
+#
+# params = HTTP::Params.encode({ "q" => "test"}) # => q=test
+# response = HTTP::Client.get "http://www.example.com?" + params
```

Comment:
Maybe worth having a more complex example? 2 parameters, one of which contains something that needs to be escaped? `{"author" => "John Doe", "offset" => 20}`

Reply:
Ok, those sound good to me.

Reply:
Although I had to change the second value to a string instead of an int. "20" works, but 20 throws an error.